content/en/docs/contribute/style/style-guide.md

```diff
@@ -42,12 +42,9 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 ## Documentation formatting standards
 
-### Use camel case for API objects
+### Use upper camel case for API objects
 
-When you refer to an API object, use the same uppercase and lowercase letters
-that are used in the actual object name. Typically, the names of API
-objects use
-[camel case](https://en.wikipedia.org/wiki/Camel_case).
+When you refer specifically to interacting with an API object, use [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as Pascal Case. When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).
```

Comment:
> When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).

This isn't quite the approach I usually take. I use sentence-style capitalization **only** when the `kind` of the resource is also an word or bit of IT jargon in the target localization.

**Yes:**
- secret (the English word “secret”)
- namespace (the English word “namespace”)
- Namespace (meaning the `kind` of an object in the API)
- nœud (_in French text_)
- node (_in English text_)
- Node (in any localization - and, again, meaning the `kind` of an object in the API)
- PersistentVolumeClaim (again meaning the `kind` of an object in the API)
- Secret (again meaning the `kind` of an object in the API)
- volume (because you can't `kubectl get Volume` but you can, eg, `kubectl get Secret`)
- certificate signing request (common IT jargon outside of a Kubernetes context)
- CertificateSigningRequest (again, meaning the `kind` of an object in the API)
- CSR (might need expanding if the first use, but subsequent uses are fine)
- PVC (might need expanding if the first use, but subsequent uses are fine)

**No:**
- configmap
- poddisruptionbudget
- tokenreview
- persistentvolume
- persistentVolume (should be PersistentVolume)
- statefulset

I also **advise against** _phrases_ (multiple words) that exactly match the way you would say an API name:
- pod disruption budget
- persistent volume claim
- config map (even “configuration map” would be a bit unidiomatic)
- subject access review
- stateful set
- daemon set

However, because it's a different pronunciation, I think it's fine to write:
- &hellip;implements horizontal Pod autoscaling&hellip;
- Kubernetes persistent storage&hellip;
- &hellip;according to the hisruption budget&hellip;
- &hellip;review subject access to&hellip;
- &hellip;signing the CSR&hellip;

It's not 100% clear cut. There are nuances. If you want to write, eg, “a manual token review process would take too much time” then I would never suggest rewriting that because “TokenReview” sounds the same. _The whole point of these different capitalization styles is to help the reader tell these cases apart_.